<?xml version="1.0"?>

<launch>

	<arg name="config" default="parameters.yaml"/>
	<arg name="bt_list" default="bt_list.yaml"/>
	<arg name="num_robots" default="1"/>

	<node pkg="simulator" type="show_tree.py" name="show_tree" output="screen">
		<param name="config" type="string" value="$(arg config)" />
		<param name="bt_list" type="string" value="$(arg bt_list)" />
		<param name="robot_id" type="int" value="0" />
		<param name="num_robots" type="int" value="$(arg num_robots)" />
		<param name="seed" type="int" value= "1" />
	</node>

</launch>